### DISCUSSION TITLE
Android Mining not working

### DISCUSSION THREAD

**Discussion by Raabitco** [revoxhere/duino-coin#869] [Q&A]
Hi everyone, Since long time android app for mining are not working, is there any update regarding that?

**Comment by YeahNotSewerSide** [comment]
@Raabitco Android miner is not maintained as an official part of duino coin, please consider opening issue in miner's repository, meanwhile try new release 
https://github.com/DoctorEenot/DUCOAndroidMiner/releases 
all connection issues were fixed in last release.

**Comment by Raabitco** [comment]
Thank you for your reply, I do appreciate it.
